protocol/avdl/stellar1/local.avdl

```diff
@@ -382,6 +382,8 @@ protocol local {
     boolean pending;
   }
 
+  SendPaymentResLocal sendPathLocal(AccountID source, string recipient, PaymentPath path, string note, string publicNote);
```

Comment:
then `sendPaymentLocal` could be reused with a protocol addition of `union { null, PaymentPath } path`